general/fastq/methylated_dna/bisulfite-seq_test_data.md

```diff
@@ -0,0 +1,61 @@
+### E. coli paired-end test dataset for Bisulfite-seq applications
```

Comment:
Looks like you copied this file here instead of moving it from the docs. Also only the docs version has the amendments (eg. link to Sherman).